packages/valory/connections/abci/connection.py

```diff
@@ -489,7 +489,7 @@ def init(self) -> None:
         cmd = self._build_init_command()
         subprocess.call(cmd)  # nosec
 
-    def start(self, start_monitoring: bool = True) -> None:
```

Comment:
I think we need to revert deployments/Dockerfiles/localnode/tendermint.py as well.

Reply:
sorry I did not mean to undo [e7d6981](https://github.com/valory-xyz/open-autonomy/pull/1082/commits/e7d6981c8ff115470696cf8f63e9ebdd1eeb708e); restoring `True`